instance/github.py

```diff
@@ -86,12 +86,12 @@ def get_settings_from_pr_body(pr_body):
         return ''
 
 
-def get_pr_by_number(fork_name, pr_number):
+def get_pr_by_number(pr_target_repo, pr_number):
     """
     Returns a PR object based on the reponse
     """
-    r_pr = get_object_from_url('https://api.github.com/repos/{fork_name}/pulls/{pr_number}'.format(
-        fork_name=fork_name,
+    r_pr = get_object_from_url('https://api.github.com/repos/{pr_target_repo}/pulls/{pr_number}'.format(
+        pr_target_repo=pr_target_repo,
```

Comment:
@Kelketek Good idea to make the names more explicit here. To keep variable names consistent, maybe `pr_target_fork_name`? It might not be the best of names, but currently `fork_name` is used consistently across the code base to mean a combination of github org name and repo name - ie `org/repo`.